windows_containers/understanding-windows-container-workloads.adoc

```diff
@@ -10,7 +10,9 @@ toc::[]
 
 [NOTE]
 ====
-Multi-tenancy for clusters that have Windows nodes is not supported. Hostile multi-tenant usage introduces security concerns in all Kubernetes environments. Additional security features like link:https://kubernetes.io/docs/concepts/policy/pod-security-policy/[pod security policies], or more fine-grained role-based access control (RBAC) for nodes, make exploits more difficult. However, if you choose to run hostile multi-tenant workloads, a hypervisor is the only security option you should use. The security domain for Kubernetes encompasses the entire cluster, not an individual node. For these types of hostile multi-tenant workloads, you should use physically isolated clusters.
+Multi-tenancy for clusters that have Windows nodes is not supported. Clusters are considered _multi-tenant_ when multiple workloads operate on shared infrastructure and resources. If one or more workloads running on an infrastructure cannot be trusted, the multi-tenant environment is considered _hostile_.
+
+Hostile multi-tenant usage introduces security concerns in all Kubernetes environments. Additional security features like link:https://kubernetes.io/docs/concepts/policy/pod-security-policy/[pod security policies], or more fine-grained role-based access control (RBAC) for nodes, make exploits more difficult. However, if you choose to run hostile multi-tenant workloads, a hypervisor is the only security option you should use. The security domain for Kubernetes encompasses the entire cluster, not an individual node. For these types of hostile multi-tenant workloads, you should use physically isolated clusters.
```

Comment:
One small suggestion:
Additional security features like pod security policies, or more fine-grained role-based access control (RBAC) for nodes, make exploits more difficult.
->
Additional security features, like pod security policies or more fine-grained role-based access control (RBAC) for nodes, make use more difficult.
![Screenshot from 2024-02-20 10-52-02](https://github.com/openshift/openshift-docs/assets/89447174/9966dc2e-64d9-4327-b5ab-f648fc3394c8)

Reply:
@smunje1 Thanks for your thoughtful review. In this instance,  _exploit_ is used as a noun.

[noun](https://duckduckgo.com/?q=exploit+meaning&ia=web)
A program or system designed to take advantage of a particular error or security vulnerability in computers or networks.

However, you pointed out a possible translation issue. I can find a better word.